### DISCUSSION TITLE
مسدود کردن سایت های ایرانی

### DISCUSSION THREAD

**Discussion by amirgi73** [Gozargah/Marzban#300] [Q&A] [answered]
من طبف [این دیسکاشن](https://github.com/Gozargah/Marzban/discussions/38) فایل کانفیگ xray رو ادیت کردم. منتهی این کار باعث میشه که کانفیگ‌های نودهام کار نکنن. یعنی در ال حاضر من یک سرور مستر دارم و 2 تا نود. بعد اعمال این تغییرات کانفیگ‌های سرور مستر کار میکنه. ولی کانفیگ‌های نودها هیچکدوم کار نمیکنه و ارور EOF میگیرم توی کلاینت. ضمنا من فقط از ریالیتی استفاده میکنم. و پنل هم فولی سینگل پورت و آخرین ورژن stable هستش.
فایل کانفیگ جهت بررسی ضمیمه شد. 
[xray_config.zip](https://github.com/Gozargah/Marzban/files/11618652/xray_config.zip)

**Comment by ghost** [comment, marked as answer]
سلام
مراحل یک و دو رو روی سرور نودهاتون هم انجام بدین.

**Reply by RenasPasha** [reply]
خب تو مرحله2 باید فایل env. رو ویراش کنیم و مقدار  "/XRAY_ASSETS_PATH = "/var/lib/marzban/assets  رو بهش اضافه کنیم
اونوقت باید رو سرور نودها  پنل مرزبان رو نصب کنیم دیگ
درسته؟؟
